### DISCUSSION TITLE
"x-nextjs-cache" HTTP Header

### DISCUSSION THREAD

**Discussion by marco910** [vercel/next.js#36388] [Help] [answered]
Hi community,

I'm using Next.js version 12.1.1 on getting in production an HTTP header`x-nextjs-cache`. Where does this come from, what does it mean, and is it possible to disable it?

Thanks for your help!

**Comment by ijjk** [comment, marked as answer]
Hi, this is documented here https://nextjs.org/docs/api-reference/data-fetching/get-static-props#revalidate and here https://nextjs.org/docs/api-reference/next/image#caching-behavior

**Comment by marco910** [comment]
Hi @ijjk 
Thanks! That is what I was looking for.

**Reply by Tikam02** [reply]
I'm Getting cache miss 

X-Nextjs-Cache: MISS


how to to fix it to HIT

X-Nextjs-Cache: HIT

**Reply by ijjk** [reply]
If it is deployed on Vercel it is expected to always be `MISS` and the header is no longer set on Vercel. Locally for it to be a `HIT` you must be using `getStaticProps` for a page and it already be generated.
